packages/boxel/addon/components/boxel/expandable-banner/index.css

```diff
@@ -80,6 +80,15 @@
   outline: transparent;
 }
 
+.boxel-expandable-banner__details[open]:focus-within {
+  outline: var(--boxel-outline);
+  outline-offset: -4px;
+}
```

Comment:
I considered this solution, but this means that adding links or other interactive content inside this will be potentially misleading. This can work when we have good support for `:has` to identify that the focus is on the right element, as progressive enhancement.

Reply:
Safari already has support. Chrome intends to ship this in [105](https://groups.google.com/a/chromium.org/g/blink-dev/c/bRsbl3wLuyk/m/mt3iSKNHBQAJ?pli=1).

I've been holding off on this issue since the way the interactivity is presented is technically correct, despite not being pretty, and a good CSS fix for it has been on the horizon for a while.

Reply:
For this issue, I would wait till Chrome 105, use the `:has` pseudoselector to fix this + make sure that it's working correctly.

Reply:
OK, sounds good. Closing and I noted this in the Linear ticket.